config/application.yml.default

```diff
@@ -132,6 +132,8 @@ idv_acuant_sdk_upgrade_a_b_testing_enabled: false
 idv_acuant_sdk_upgrade_a_b_testing_percent: 50
 idv_send_link_attempt_window_in_minutes: 10
 idv_send_link_max_attempts: 5
+idv_tmx_test_csp_disabled_emails: '[]'
+idv_tmx_test_js_disabled_emails: '[]'
```

Comment:
I would have expected default for JSON to be '{}' rather than '[]', but it looks like it works fine.

Reply:
If I were to use `{}` then it would be parsed into a hash where we would expect an array. An empty array is valid JSON so I went with that to ensure we consistently get an array.